packages/backend/src/storage/storage.ts

```diff
@@ -440,10 +447,38 @@ export class Storage {
 
     const writeStream = fs.createWriteStream(filePath)
 
+    let downloadedBytes = 0
+    let stopwatch = 0
+
     for await (const entry of entries) {
       await new Promise<void>((resolve, reject) => {
         writeStream.write(entry, err => {
+          // Do not proceed with error
           if (err) reject(err)
+
+          let transferSpeed = 0
+
+          if (stopwatch === 0) {
+            stopwatch = Date.now()
+          } else {
+            const timestamp = Date.now()
+            const delay = (timestamp - stopwatch) * 1000 // in seconds
```

Comment:
I think we should divide, not multiply if we want seconds?

Reply:
true